src/styles/search.css

```diff
@@ -0,0 +1,33 @@
+.search-input {
+    width: 100%;
+    height: 50px;
+    border-radius: 100px;
+    font-size: 20px;
+    padding: 0 10px;
+    background-color: #BF0016;
+    border-color: white;
+    color: white;
+    font-weight: bold;
+    font-style: italic;
+}
+
+.search-button {
+    border-radius: 100px;
+    background-color: #BF0016;
+    font-weight: bold;
+    font-size: 20px;
+    border-color: white;
+    color: white;
+
+}
+
+.search-button:hover {
+    background-color: black;
+    color: gold;
+    font-style: italic;
+}
+
+form {
```

Comment:
Rather than targeting the `form` element itself, it would be better here to use the `search-form` class you've already added 

Also, the `flex-direction: row` is already applied as it is the default for flex, so this can be removed